far/FarEng.hlf.m4

```diff
@@ -2295,6 +2295,7 @@ $ #Special symbols#
     #!&~~#      List of selected short file names
     #!:#       Current drive in the format "C:"
              For remote folders - "\\\\server\\share"
+    #!*#       Description of the current file
```

Comment:
Could you please add the following line to FarRus.hlf.m4:
```
#!*#       Описание файла
```

Reply:
Of course, thanks a lot. I did not know the exact translation.